src/affordance/tasks/sequential_interpreter.py

```diff
@@ -43,10 +43,12 @@ def program_ends(program: str) -> bool:
         """Return true if the program contains [EOQ] and a final answer."""
         return "[EOQ]\nAns:" in program
 
-    def batch_visit(self, prefixes: List[str], programs: List[str]) -> List[str]:
+    def batch_visit(
```

Comment:
Sometimes this function is called with a batch across multiple separate evaluations, so we probably need to make `run_title` be a `List[str]`.